beetsplug/web/__init__.py

```diff
@@ -69,10 +69,14 @@ def json_generator(items, root):
             first = False
         else:
             yield ','
-        yield json.dumps(_rep(item))
+        yield json.dumps(_rep(item, expand=expand))
     yield ']}'
 
 
+def is_expand(args):
```

Comment:
Not sure, if I remove the method I would need to use the "in" operator

`
return flask.jsonify(_rep(entities[0], expand='expand' in flask.request.args))
`

This would also cause duplicate code. What do you think?